planning/planners/src/encode.rs

```diff
@@ -659,8 +659,8 @@ pub fn encode(pb: &FiniteProblem, metric: Option<Metric>) -> std::result::Result
                     ConstraintType::Sum(sum) => {
                         let sum = sum.sum.clone();
 
-                        solver.model.enforce(sum.clone().leq(0), [instance.chronicle.presence]);
-                        solver.model.enforce(sum.geq(0), [instance.chronicle.presence]);
+                        solver.model.enforce(sum.clone().leq(0), []);
```

Comment:
Suspicious : it means that the constraint must hold even if the chronicle is not there.

Reply:
The reason why I removed it, is that for the moment the linear constraints are not reified. It is not a fix but a temporary workaround.
```
panicked at 'Unsupported reified linear constraints.', ../aries/solver/src/solver/solver_impl.rs:231:21
```

Reply:
It cannot be refied but it can now be given a scope. See the `add_opt_linear_constraint` method used in solver_impl.rs:231.

Reply:
I changed the encoding to this:
```rust
solver.model.bind(sum.clone().leq(LinearSum::zero()), value);
solver.model.bind(sum.geq(LinearSum::zero()), value);
```
But it still panicks, and I suspect **value** to cause the problem. However I fail to find why.